# SensorProcessingData.msg

Header header
motion_ctrl/SequiturData sequitur
motion_ctrl/DeadReckoningData deadreck
